src/snapred/backend/recipe/GenerateFocussedVanadiumRecipe.py

```diff
@@ -0,0 +1,55 @@
+import json
+from typing import Any, Dict
+
+from snapred.backend.dao.ingredients import GenerateFocussedVanadiumIngredients as Ingredients
+from snapred.backend.log.logger import snapredLogger
+from snapred.backend.recipe.algorithm.SmoothDataExcludingPeaksAlgo import SmoothDataExcludingPeaksAlgo
+from snapred.meta.decorators.Singleton import Singleton
+
+logger = snapredLogger.getLogger(__name__)
+
+
+@Singleton
+class GenerateFocussedVanadiumRecipe:
+    """
+
+    The purpose of this recipe is to generate a focussed vandium. This recipe will take a pre-processed
+    raw vanadium workspace that has been preprocessed and then grouped via group processing recipe and
```

Comment:
You could probably remove "that has been preprocessed" since it's a bit redundant